src/styles.ts

```diff
@@ -107,4 +98,14 @@ export const esphomeDialogStyles = css`
   mwc-list.platforms mwc-list-item {
     padding: 0 24px;
   }
+
+  mwc-button {
+    --mdc-theme-primary: var(--primary-text-color);
+  }
+`;
+
+export const esphomeSvgStyles = css`
+  .svg-fill-primary {
+    fill: var(--primary-text-color);
```

Comment:
I wonder if `fill: currentColor` isn't the better way to go here, since that makes the icons colored like the text.

Reply:
this can be fixed in a follow-up PR. Let's get this in the beta.